treeherder/etl/classification_loader.py

```diff
@@ -110,12 +112,18 @@ def autoclassify_failures(self, failures):
             )
             raise
 
-        for tasks in failures.values():
+        for group, tasks in failures.items():
```

Comment:
is group the manifest name, path, or path/testcase_filename ?

Reply:
The path I think, I'm not sure it's the right term, but [here](https://community.taskcluster-artifacts.net/MQUn3pgBSsmTbbTI5Ulw1A/0/public/classification.json) you have one artifact that could be retrieved through Treeherder, `group` will be `testing/web-platform/tests/webdriver/tests/new_session` for this specific case

Reply:
ok, so group is the manifest.  I see:
https://community.taskcluster-artifacts.net/EGXVIcuOSSW0-qCPu7Uo8w/0/public/classification.json

has:
`dom/serviceworkers/test/mochitest.ini:dom/serviceworkers/test/mochitest-common.ini`

we will have a test case name instead, something like: `dom/serviceworkers/test/mochitest.ini:dom/serviceworkers/test/test_dosomething.js`

so if we are matching on the `{group}` in:
`Bugscache.objects.get(summary__endswith=f"{group} | single tracking bug")`

this will not find anything.  

does the `failures` data have specific line numbers?  Should we revisit the bug titles?

Reply:
Once https://github.com/mozilla/mozci/issues/685 and https://github.com/mozilla/mozci/issues/700 are done and merged, I will be able to refactor this logic to match the expected one. :smile: